extras/parse_travis_log.py

```diff
@@ -177,6 +180,35 @@ def process_static_analysis(f, line):
         if line.startswith(' - clang-format for '):
             d.update(process_clang_format(f, filename))
 
+def process_pep8(f, line):
+    """
+    Process PEP8 output for a certain file.
+    """
+    # Check PEP8 on file pynest/nest/lib/hl_api_connections.py:
+    filename = line.split(' ')[-1].strip()[0:-1]
+    d = set()
+    res = {filename: d}
+    while True:
```

Comment:
@apeyser I structured the script with the `readline()` method. When I replace it with the iterator, I get errors like:

    ValueError: Mixing iteration and read methods would lose data

As this is not directly related to PEP8, I would like to put this into another issue, and here just document, what the lines are intended to do.